plugin-extending.md

```diff
@@ -311,4 +311,8 @@ Similarly we can remove the menu items with the same event:
         $manager->removeMainMenuItem('October.Cms', 'cms');
         $manager->removeSideMenuItem('October.Cms', 'cms', 'pages');
 
+        $manager->removeSideMenuItems('October.Cms', 'cms', [
+            'pages',
+            'cms'
```

Comment:
Is cms a side menu item in this context?

Reply:
Yes

Reply:
I meant is it a real item, not just an example: https://github.com/octobercms/october/blob/develop/modules/cms/ServiceProvider.php#L120-L166. It is not.

Reply:
@LukeTowers How would you want me to change it?

Reply:
@4nik3t just change the side menu references to be actual side menu items present under the October.Cms cms main menu item so that it's more accurate.

Reply:
@LukeTowers  I have replaced cms with partials